pyproject.toml

```diff
@@ -26,6 +26,11 @@ requires-python = ">= 3.10"
 [tool.setuptools.dynamic]
 dependencies = {file = ["requirements.txt"]}
 
+[project.optional-dependencies]
+vllm = [
```

Comment:
hm, what would happen to imports from vllm if this optional dependency is not installed? why is this optional?

Reply:
I feel optional is better, I don't want to add more dependencies unless needed.

Reply:
this is good, thanks